purview/utils/general.py

```diff
@@ -194,3 +202,38 @@ def query_neo4j(query):
                 if serialized_node not in results_dict[node_label]:
                     results_dict[node_label].append(inflated_node.serialized)
     return results_dict
+
+
+def get_corelated_nodes(results, count=True):
+    """
+    Create a raw cypher query for story nodes and get corelated nodes or their count.
+    """
+    # To avoid circular imports
+    from purview.models import story_flow
+
+    queries_dict = {}
+    for artifact in results:
+        query = 'MATCH '
+        if story_flow[artifact[0]]['backward_label']:
+            node = node_query(artifact[0], story_flow[artifact[0]]['uid_name'], artifact[
+                              1][story_flow[artifact[0]]['uid_name']])
+            next_node = node_query(story_flow[artifact[0]]['backward_label'])
+            query += '{0}-[:{1}]-{2}\n'.format(node, story_flow[artifact[0]]
```

Comment:
Can you assign `story_flow[artifact[0]]['backward_relationship'][:-1]` to a variable for clarity?